fth/t_tools.fth

```diff
@@ -63,6 +63,18 @@ CREATE the-test 128 CHARS ALLOT
     LOOP \ save them
 ;
 
+
+: int>str s>d swap over dabs <# #s rot sign #> ;
```

Comment:
Can we use (.) instead? 

    see (.)

Reply:
Happy to.

Is this defined in a standard anywhere?
If not I wonder where would be the best place to document it.